applications/incompressible_navier_stokes/periodic_hill/application.h

```diff
@@ -330,6 +358,22 @@ class Application : public ApplicationBase<dim, Number>
           "Manifolds might not be applied correctly for TriangulationType::FullyDistributed. "
           "Try to use another triangulation type, or try to fix these limitations in ExaDG or deal.II."));
 
+      unsigned int const manifold_id = 111;
```

Comment:
why was the manifold not here? I am sure I did not add this.

Reply:
note that this might also impact the stability of restart: we do have the option to store the mapping and restore it, but my tests in connection to https://github.com/exadg/exadg/pull/783 used the manifold only.
I will test once it is clear if I should remove the manifold here.

Reply:
Regarding the manifold: I think we should remove all manifold descriptions, the triangulations should be a rectangular brick. It could very well be that I forgot some remainders in the code as I did not closely check that, so cleaning up this makes sense. And then we should only use the mapping to apply the deformation.

Reply:
I fear if we dont hit the exact geometry, all the measures downstream will be off :cry:

Reply:
ok I tested with higher resolution --- polynomial degree 9.
<img width="1414" height="892" alt="image" src="https://github.com/user-attachments/assets/d03ca294-6821-421e-9fa7-d35da1bd30b0" />
sorry for the false alarm!
I will push the changes = removing that manifold that is attached in the current branch.
Then we can merge.